versions.txt

```diff
@@ -7,3 +7,4 @@ mcagar@9.0.0
 alloy@14.0.0
 bridge@4.7.0
 sugar@9.3.0
+snooker@11.1.1
```

Comment:
Does this need to be a minor since this is a bug fix only it could go into a patch.

Reply:
Oh right, is the current version of snooker not already 11.1.0?
https://github.com/tinymce/tinymce/blob/693fef6287304759bc2f3dc31f76e9e74a68d8d1/modules/snooker/package.json#L4
Or am I misunderstanding?

Reply:
Bumped snooker version to 11.2.1 now after the 6.8 release